src/sdk/pynni/nni/networkmorphism_tuner/networkmorphism_tuner.py

```diff
@@ -34,7 +36,35 @@
 
 
 class NetworkMorphismTuner(Tuner):
-    """NetworkMorphismTuner is a tuner which using network morphism techniques."""
+    """
+    NetworkMorphismTuner is a tuner which using network morphism techniques.
+    
+    Attributes
+    ----------
+    n_classes : int
+        The class number or output node number (default: {10})
+    input_shape : tuple
+        A tuple including: (input_width, input_width, input_channel)
+    t_min : float
+        The minimum temperature for simulated annealing. (default: {Constant.T_MIN})
+    beta : float
+        The beta in acquisition function. (default: {Constant.BETA})
```

Comment:
suggest using ``Constant.BETA``, and It will be rendered like this: 
![image](https://user-images.githubusercontent.com/3284327/68113225-3b4d2200-ff2e-11e9-8e15-8b30a5ef1620.png)

Also suggest you applying this trick elsewhere.

Reply:
Good suggestion.